exampleCourse/questions/gallery/multipleChoice/complex/server.py

```diff
@@ -17,7 +17,7 @@ def generate(data):
     v0x = v0 * math.cos(theta * math.pi / 180)
     v0y = v0 * math.sin(theta * math.pi / 180)
     # time in the air (s)
-    t = round(d / v0x)
+    t = round(d / v0x, 3)
```

Comment:
My only question is if it makes more sense to round this after we compute the height of the cliff (that is, move the rounding to the line with the `data["params"]["t_c"] = ...` assignment). That would ensure that the height is computed with the full-precision value.

Reply:
agree with that. I actually fixed another version of this question using this same method proposed above, but forgot that the question also existed in XC 101.

Reply:
Gotcha, thanks for confirming that this is the way to go! @ZacWarham do you want to get this fixed up? If I don't hear from you in a day or too, I'll make the change myself.

Reply:
Can do @nwalters512 - I just finished up my thesis so have had a couple days away from the computer